packages/rx_bloc/lib/src/extensions.dart

```diff
@@ -109,7 +109,7 @@ extension Bind<T> on Stream<T> {
   /// Be aware that the binding is facilitating the subscribing, so the
   /// unsubscribing needs to be handled accordingly either by
   /// using [CompositeSubscription] or manually.
-  StreamSubscription<T> bind(BehaviorSubject<T> subject) =>
+  StreamSubscription<T> bind(BehaviorSubject<T?> subject) =>
```

Comment:
@PifeVoxd, can you please elaborate on this?
To me this code looks like I'm forced to use that binder only to the values that are nullable, is that a correct statement? What if I want to use it to nin-nullable values ?

Reply:
No, it's not a correct statement, however I've looked further into the issue I was having with the previous implementation, and actually there are some quite interesting and complex things happening here, suffice it to say neither the previous nor this implementation are statically null safe, and there is a case (a case which is quite common) where both implementations end up throwing runtime type errors. I will look into it  further.